Library/Formula/gcc47.rb

```diff
@@ -122,15 +117,8 @@ def install
       # don't wander around telling little children there is no Santa
       # Claus.
       "--enable-version-specific-runtime-libs",
-      "--enable-libstdcxx-time=yes",
```

Comment:
I guess we probably need to revbump it if removing it will also change compat?

Reply:
Yes, you're right.